cpu/pic32/cfs-coffee-arch.c

```diff
@@ -47,7 +48,7 @@ pic32_flash_erase(coffee_page_t sector)
   flash_erase(erase_addr);
 
   for(pos = 0; pos < COFFEE_SECTOR_SIZE; pos+=4) {
-      flash_write_word(erase_addr + pos, 0);
+      flash_write_word((char*)erase_addr + pos, 0);
```

Comment:
There is no difference between using ```uint8_t*``` and ```char*```. Both will always have a size of 1, so the address will be identical.